test/pretty.test.js

```diff
@@ -8,6 +8,12 @@ const writer = require('flush-write-stream')
 const { once } = require('./helper')
 const pino = require('../')
 
+const isWin = process.platform === 'win32'
+// skip tests on Windows as colour codes are different and tests fail
+if (isWin) {
```

Comment:
I think you should emit a tap comment here to say why this file is being skipped.

I think the API is tap.comment(str)

Reply:
Addressed!